src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,28 @@
+package pages;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+public class HomePage extends BasePage {
+
+    static WebDriver driver;
+    WebDriverWait wait;
+    By userAvatarIcon = By.cssSelector("img.avatar");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```